projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.tsx

```diff
@@ -1,13 +1,13 @@
 import { Component, Prop, Event, EventEmitter } from '@stencil/core';
-import { DotKeyValueField, DotLabel } from '../../../models';
-import { getTagLabel } from '../../../utils';
+import { DotKeyValueField } from '../../../models';
 
 @Component({
     tag: 'key-value-table'
 })
 export class KeyValueTableComponent {
     @Prop() items: DotKeyValueField[] = [];
     @Prop() disabled = false;
```

Comment:
doc?

Reply:
When we start doing test and stuff, let's make sure we do docs, params and tests in each component we assigned to.